### DISCUSSION TITLE
Can import be modified?

### DISCUSSION THREAD

**Discussion by wsongah** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#45] [Q&A]
Can we add import ourselves? At Least, can we add "import java.util.ArrayList;" ? It seems quite necessary in some files!

**Comment by Derppening** [comment]
[FAQ](https://github.com/CastleLab/COMP3021-2021Fall-PA1-Student-Version#faq)

![image](https://user-images.githubusercontent.com/4293438/135719666-7d3483d3-c65e-4418-93d7-1e0344461df5.png)
